tensorflow/lite/kernels/reverse.cc

```diff
@@ -52,10 +51,21 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
     return kTfLiteError;
   }
 
-  // TODO(renjieliu): support multi-axis case.
-  if (NumElements(axis) > 1) {
-    context->ReportError(context, "Current does not support more than 1 axis.");
-  }
+  // Allocate temporary tensor which is required in kernel operation
```

Comment:
Does the TF implementation require a temporary/intermediate tensor?

Reply:
> Does the TF implementation require a temporary/intermediate tensor?

TF implementation rely on changing only view of Tensor, rather than Tensor itself. But in TFLite it need to be reflected on output buffer directly. So i think maintaining a copy Tensor is required here. Please let me know your valuable opinion, Thanks!